doc/game.md

```diff
@@ -36,81 +107,58 @@ main() {
+# GameLoop
 
-It is important to notice that `Game` is an abstract class with just the very basic implementations of the gameloop.
+The `GameLoop` module is a simple abstraction over the game loop concept. Basically most games are
+built upon two methods:
 
-As an option and more suitable for most cases, there is the full-featured `BaseGame` class. For example, Forge2D games uses `Forge2DGame` class;
+ - The render method takes the canvas for drawing the current state of the game.
+ - The update method receives the delta time in seconds since the last update and allows you to move
+  to the next state.
 
-The `BaseGame` implements a `Component` based `Game` for you; basically it has a list of `Component`s and passes the `update` and `render` calls appropriately. You can still extend those methods to add custom behavior, and you will get a few other features for free, like the passing of `resize` methods (every time the screen is resized the information will be passed to the resize methods of all your components) and also a basic camera feature. The `BaseGame.camera` controls which point in coordinate space should be the top-left of the screen (defaults to [0,0] like a regular `Canvas`).
+The `GameLoop` is used by all of Flame's `Game` implementations.
 
-A `BaseGame` implementation example can be seen below:
+# Flutter Widgets and Game instances
 
-```dart
-    class MyCrate extends SpriteComponent {
-
-        // creates a component that renders the crate.png sprite, with size 16 x 16
-        MyCrate() : super.fromSprite(16.0, 16.0, new Sprite('crate.png'));
-
-        @override
-        void onGameResize(Size size) {
-            // we don't need to set the x and y in the constructor, we can set then here
-            this.x = (size.width - this.width) / 2;
-            this.y = (size.height - this.height) / 2;
-        }
-    }
-
-    class MyGame extends BaseGame {
-        MyGame() {
-            add(new MyCrate()); // this will call resize the first time as well
-        }
-    }
-```
-
-To remove components from the list on a `BaseGame` the `markToRemove` method can be used.
-
-## Flutter Widgets and Game instances
+Since a Flame game can be wrapped in a widget, it is quite easy to use it alongside other Flutter
+widgets. But still, there is the Widgets Overlay API that makes things even easier.
 
-Since a Flame game can be wrapped in a widget, it is quite easy to use it alongside other Flutter widgets. But still, there is a the Widgets overlay API that makes things even easier.
-
-`Game.overlays` enables to any Flutter widget to be shown on top of a game instance, this makes it very easy to create things like a pause menu, or an inventory screen for example.
+`Game.overlays` enables to any Flutter widget to be shown on top of a game instance, this makes it
+very easy to create things like a pause menu, or an inventory screen for example.
```

Comment:
```suggestion
very easy to create things like a pause menu or an inventory screen.
```

(like already means it's a list of examples)

Reply:
I haven't written the sentence, but I think it flows a lot better with `for example` in the end, and it underpins the point that pretty much anything could be done with the widgets and that these two things are just examples of how widgets are commonly used.